ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -196,6 +196,36 @@ def test_tensorflow_solve(
     )
 
 
+# logdet
+@handle_cmd_line_args
+@given(
+    dtype_and_x=_get_dtype_and_matrix(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.logdet"
+    ),
+)
+def test_tensorflow_logdet(
+    dtype_and_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
```

Comment:
`fw=fw` needs to be removed from here according to the new policy.

Reply:
Done!
(is this documented yet?)